documentation/src/main/asciidoc/topics/data_container/ref_eviction_strategies.adoc

```diff
@@ -0,0 +1,27 @@
+[id='eviction_strategies_{context}']
+[[eviction_strategies]]
+= Eviction Strategies
+
+`NONE`::
+Eviction is not enabled and it is assumed that the user will not invoke evict directly
```

Comment:
It's not a problem per se, but truncating lines at a specific character count messes up your diffs when you add or remove content and reflow everything. The custom is to use one sentence per line, which is fairly easily to work with and it strikes a reasonable balance between line length and diff readability.